tests/test_clusterv2.py

```diff
@@ -0,0 +1,362 @@
+from __future__ import print_function
+import sys
+import os
+if sys.version_info > (2, 7, 0):
+    import unittest
+else:
+    import unittest2 as unittest
+from mock import Mock
+import tempfile
+sys.path.append(os.path.join(os.path.dirname(__file__), '../bin'))
+import qds
+from qds_sdk.connection import Connection
+from test_base import print_command
+from test_base import QdsCliTestCase
+from qds_sdk.cloud.cloud import Cloud
+
+class TestClusterCreate(QdsCliTestCase):
```

Comment:
Please add test cases for other cluster actions like `list`, `terminate`, `add_node` etc